types/pem/tsconfig.json

```diff
@@ -12,8 +12,8 @@
         "typeRoots": [
             "../"
         ],
-        "types": [],
         "noEmit": true,
+        "types": ["node"],
```

Comment:
**Lint error:** tsconfig.json `types` array must be empty. Use triple-slash ref in `index.d.ts` instead:

    /// <reference types="node" />

Reply:
thanks for pointing out this, moved node types reference to index.d.ts